server/README.md

```diff
@@ -0,0 +1,44 @@
+# cardano-browser-tx-server
+
+This is a small Haskell project to provide services to the `cardano-browser-tx` frontend that cannot be achieved using Purescript
```

Comment:
Why can't they be achieved using Purescript? Is it actually the goal of this project to provide only such services?

Reply:
There are some things that we can't do very easily with PS. The fee estimation we could probably do by replicating `evaluateTransactionFee` and including the necessary information from Alonzo `ProtocolParameters`. There are others that we can't do easily though, like Plutus' `applyCode`